python/plugins/processing/algs/gdal/ogrinfo.py

```diff
@@ -48,6 +51,13 @@ def initAlgorithm(self, config=None):
                                                         self.tr('Suppress metadata info'),
                                                         defaultValue=False))
 
+        options_param = QgsProcessingParameterString(self.OPTIONS,
```

Comment:
What kind of options are you going to use here? `ogrinfo` does not produce any output except raw HTML file. Probably you goal was to allow use additional command-linre options? In the latter case I'd suggest to rename parameter and reword its desctription  to be consistent with other algorithms.

Also it would be nice to add tests.